pyHS100/smartplug.py

```diff
@@ -80,6 +80,54 @@ def state(self, value: str):
         else:
             raise ValueError("State %s is not valid.", value)
 
+    @property
+    def brightness(self) -> Optional[int]:
+        """
+        Current brightness of the device, if supported
+
+        :returns: integer
+        :rtype: int
+        
+        """
+        if not self.is_dimmable:
+            return None
+
+        return int(self.sys_info['brightness'])
+        
+    @brightness.setter
+    def brightness(self, value: int):
+        """
+        Set the new switch brightness level
+
```

Comment:
Maybe it is worth mentioning here, that this will turn the light on if it isn't already on?